k8s/src/test/scala/io/buoyant/k8s/ThirdPartyResourceTest.scala

```diff
@@ -0,0 +1,102 @@
+package io.buoyant.k8s
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonSubTypes, JsonTypeInfo}
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.io.Buf
+import com.twitter.util.Future
+import io.buoyant.k8s.{KubeObject => BaseObject}
+import io.buoyant.test.Awaits
+import org.scalatest.FunSuite
+
+/**
+ * This test both exercises the versioning functionality in `io.buoyant.k8s`, and provides
+ * an example implementation. (See `namerd` for the current usage in the Buoyant suite of projects).
+ */
+class ThirdPartyResourceTest extends FunSuite with Awaits {
+  import ThirdPartyResourceTest._
+
+  object Fixtures {
+    val bookList = Buf.Utf8("""{"kind":"BookList","apiVersion":"buoyant.io/vTest","metadata":{"selfLink":"/apis/buoyant.io/vTest/namespaces/test/books"},"items":[{"metadata":{"name":"programming-in-scala","selfLink":"/apis/buoyant.io/vTest/namespaces/test/books/programming-in-scala"},"title":"Programming in Scala","author":"Martin Odersky"}]} """)
+  }
+
+  test("namespaced: get books list") {
+    import Fixtures._
```

Comment:
Why not define booklist within the context of ThirdPartyResourceTest, instead of putting it into a separate object that needs to be imported?

```
class ThirdPartyResourceTest extends FunSuite with Awaits {
  import ThirdPartyResourceTest._

  val bookList = Buf.Utf8("""{"kind":"BookList"...

  test("namespaced: get books list") {
    val service = Service.mk[Request, Response] { req =>
```